src/main/jbake/content/docs/what.adoc

```diff
@@ -0,0 +1,170 @@
+:jbake-type: page
+:jbake-status: draft
+
+
+== Why Hawkular?
+
+.NOTE: WHY: what is the purpose, why Hawkular
+
+To be the best tool to monitor JBoss and other Java Middleware helping
+you spending less and more enjoyable time on middleware management.
+
+OR:
+
+Hawkular will reduce the effort and time spent on systems management and monitoring
+and make this task more enjoyable.
+
+
+.NOTE: How
+
+This is done through automation of various aspects like resource discovery, root cause analysis,
+predictive analytics and easy to use user interfaces.
+
+
+== The Goals of Hawkular
+
+Hawkular is clearly a project to best monitor and manage JBoss Projects as well as other Java processes.
+
+For managed objects that Hawkular knows about, it will provide a specific, _easy to use_ environment.
+Hawkular will center around the notion of _applications_ that is the unit of interest in an operating environment.
+
+Of course the tool can only be as good as the understanding of the managed objects, so it will use
+_graceful degradation_ to provide a very good experience for well understood targets and a more generic
+one for others. Managed platforms will be able to supply parts of the user interface to also provide a specific
+interface.
+
+Hawkular will be easy to integrate into existing tools landscapes. This includes access in various programming
+languages, which is especially important as many users prefer languages like Shell, Perl or Python for system
+administration work.
+
+=== Application centric
+
+Running an application and keeping it running in an optimal fashion is the main driver for IT. Which is why Hawkular
+puts the application at the center of its efforts. Applications are of course composed from individual connected
+parts, that Hawkular deals with as well.
+
+image::img/mission/app-view.png[alt=Example App]
+
+In the above example, the app consists of 3 individual deployments that access a shared database and that are linked
+together via a load-balancer. The little plus sign at the right of the database shows that there are other
+applications linked to that same database.
+
+=== Shorter Term Goals
+
+If you look at the goals, then you will clearly see that some are more near term and others need more work
+and research. Longer term goals are listed below.
+
+==== Specific UI
+
+Monitoring for resources like WildFly servers or URLs will only contain elements that are key for those resources.
+Distractions will be kept away. To set up alerts (see next section), only valid options will be displayed to allow
+for an intuitive workflow.
+
+
+==== Powerful Alerting
+
+Alerting is a core piece of a monitoring and management system. Failure states can only be fixed if someone can
+learn about them and can thus start counter measures.
+In order for notifications to happen a powerful way of expressing rules needs to be available (but in the same time
+keeping a simple and easy to use user interface).
+
+A rule could be
+
+--
+  alert me if
```

Comment:
Love the examples! Its concrete and tells me exactly what this thing can do for me.